docs/tutorials/monero.md

```diff
@@ -0,0 +1,61 @@
+Monero
+==================
+
+## Connect to remote nodes.
+
+To connect Monero nodes, you have to configure the wallet software:
+
+In [Feather wallet](http://rwzulgcql2y3n6os2jhmhg6un2m33rylazfnzhf56likav47aylq.b32.i2p/): File -> Settings -> Network -> Proxy > i2p, with this option all network traffic will be routed over i2p, including websocket connection (if enabled).
+
+## Connection between other I2P nodes.
+
+1) Configure i2pd to create server type tunnel and put in your `./tunnels.conf` file:
+
+    [monero-node]
+    type = server
+    host = 127.0.0.1
+    port = 18080 # P2P port
+    keys = monero-node.dat
+
+2) Restart i2pd or send `SIGHUP` signal.
+
+3) Find your b32 address of the node.
+
+   Go to webconsole -> [I2P tunnels page](http://127.0.0.1:7070/?page=i2p_tunnels). Look for Server tunnels and you will see address like \<long random string\>.b32.i2p next to monero-node.
+
+   The node is now available in Invisible Internet. You can check the node's status by visiting \<long random string\>.b32.i2p/get_info.
+
+4) Update your monero.conf file;
+
+    anonymous-inbound=XXX.b32.i2p:18080,127.0.0.1:18080
+
+Replace XXX with your b32 address.
+
+
+## Setup I2P service on already running public node.
+
+Open wallet inteface (the "RPC") allows anyone to connect their wallets to Monero network through the node. This is useful for users who don't run their own nodes and connecting over i2p. 
+
+If you already have a node with public RPC and now want to run i2p service, follow those steps:
+
+1) Make sure the node is already configured for public use and running. (For an example you setted up `monero.conf` such as `public-node=1` with `restricted-rpc=1`)
+
+2) Configure i2pd to create server type tunnel and put in your `./tunnels.conf` file:
+
+    [monero-node-rpc]
+    type = server
+    host = 127.0.0.1
+    port = 18089 # Restricted RPC port
+    keys = monero-node.dat
```

Comment:
```suggestion
    keys = monero-rpc.dat
```

Reply:
i think its better to keep one key file to have one address instead separate addresses for RPC and P2P

Reply:
For privacy, i personally think its better to have rpc and the node uncorrelated

you _can_ use the same key for both, but imo you shouldn't share your p2p address